pyproject.toml

```diff
@@ -54,6 +54,7 @@ exclude = '''
 
 
 [tool.isort]
-known_first_party = ["aframe", "train", "utils"]
+known_first_party = ["aframe", "train", "utils", "online", "data", "infer", "plots", "export", "ledger", "p_astro", "priors"]
+known_third_party = ["ml4gw", "amplfi", "hermes"]
```

Comment:
Do we want `ml4gw` as a known third party now that it's no longer a submodule?

Reply:
This was giving me trouble but I think you are right. I think this was b/c the ml4gw submodule folder was still in my local repo. Let me try to remove it